scripting/shavit-zones.sp

```diff
@@ -467,7 +474,8 @@ public void UnloadZones(int zone)
 				gMZ_FreestyleTypes[i] = view_as<MapZones>(-1);
 			}
 		}
-
+		
+		ClearCustomSpawn();
```

Comment:
Please make the spacing identical to the rest of the code (one new line before `return`)